src/idsk/components/header-web/header-web.js

```diff
@@ -101,26 +110,40 @@ HeaderWeb.prototype.handleLanguageSwitcherClick = function (e) {
     if(this.$activeSearch.classList.contains('idsk-header-web__brand-language--active')){
         this.$activeSearch.firstElementChild.setAttribute('aria-expanded', 'true')
         this.$activeSearch.firstElementChild.setAttribute('aria-label',  this.$activeSearch.firstElementChild.getAttribute('data-text-for-hide'))
-    }else{
+    } else {
         this.$activeSearch.firstElementChild.setAttribute('aria-expanded', 'false')
         this.$activeSearch.firstElementChild.setAttribute('aria-label',  this.$activeSearch.firstElementChild.getAttribute('data-text-for-show'))
     }
     document.addEventListener('click', this.$module.boundCheckBlurLanguageSwitcherClick, true);
 }
 
+/**
+ * Handle hide language switcher
+ * @param {object} e
+ */
+HeaderWeb.prototype.hideLanguagesList = function (e) {
+  this.$activeSearch = this.$module.querySelector('.idsk-header-web__brand-language')
+
+  toggleClass(this.$activeSearch, 'idsk-header-web__brand-language--active');
+  this.$activeSearch.firstElementChild.setAttribute('aria-expanded', 'true')
+  this.$activeSearch.firstElementChild.setAttribute('aria-label',  this.$activeSearch.firstElementChild.getAttribute('data-text-for-hide'))
+
+  document.addEventListener('click', this.$module.boundCheckBlurLanguageSwitcherClick, true);
+}
+
```

Comment:
tak isto moze ist prec lebo je to uz v develope, riesenie tohto problemu.

Reply:
Aha okay.. Tak nechápem prečo som to mal aj ja v tasku. Nevadí, dam to teda preč